CHANGELOG.rst

```diff
@@ -4,7 +4,10 @@ History
 
 0.6.6 (2020-06-11)
 ------------------
-* Improved Heading Block to accept nested text content 
+
+=======
```

Comment:
Why is this line added? It might break RST formatting.

Reply:
@shon updated 😄